sklearn/feature_extraction/tests/test_dict_vectorizer.py

```diff
@@ -76,6 +76,52 @@ def test_one_of_k():
     assert "version" not in names
 
 
+def test_iterable_value():
+    D_names = ['ham', 'spam', 'version=1', 'version=2', 'version=3']
+    X_data = [[2.0, 0.0, 1.0, 1.0, 0.0],
+              [0.0, 0.3, 0.0, 1.0, 0.0],
+              [0.0, -1.0, 0.0, 0.0, 1.0]]
+    D_in = [{"version": ["1", "2"], "ham": 2},
+            {"version": "2", "spam": .3},
+            {"version=3": True, "spam": -1}]
```

Comment:
Please add an instance with the same version repeated, to test the counting behaviour